src/analyses/varEq.ml

```diff
@@ -509,21 +509,15 @@ struct
       |> remove_reachable ~deep:false ask shallow_args
       |> remove_reachable ~deep:true ask deep_args
 
-  let safe_fn = function
-    | "memcpy" -> true
-    | "__builtin_return_address" -> true
```

Comment:
Why are we no longer treating these as safe?

Reply:
They now have `LibraryFunctions` descriptions, such that they don't need to be hardcoded into a single analysis.